keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -256,18 +257,18 @@ spec:
   secretRef: [9]
     name:
     namespace:
-  azureBlobIdentity: [10] // TODO break into auth and conn
+  protocolAzureBlob: [10] // TODO break into auth and conn
     storageAccountName:
     accountKey:
     containerName:
-  s3Identity: [11]
+  protocolS3: [11]
     endpoint:
     accessKeyId:
     bucketName:
     region:
     signatureVersion:
     userName:
-  gcsIdentity: [12]
+  protocolGcs: [12]
```

Comment:
what's the k8s convention when an acronym is embedded in a stanza name? Above or `protocolGCS`?